docs/photon/migrating-from-sequelize.md

```diff
@@ -131,19 +131,12 @@ Now in your terminal, type:
 
 ```
 cd photonjs_app
-npx prisma2 dev
+npx migrate save
```

Comment:
Note that it might not be smart/wanted to use an experimental feature here at all but find a different solution to get the DB migrated/in place.

Reply:
Maybe @nikolasburk could decide?